client/src/pages/ProjectDetails.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import IndividualProject from '../components/IndividualProject';
+
+const projectID = window.location.pathname.split('/').pop();
```

Comment:
I'd prefer you'd get the projectID from using React Router, so this properly updates when the component updates 

You can do that using a React Router hook

Reply:
Okay, changed to useParams() to get project id.